public/app/components/ChartTitle.tsx

```diff
@@ -15,8 +15,8 @@ const chartTitleKeys = {
   exceptions: 'Total number of exceptions thrown',
   unknown: '',
 
-  baseline: 'Baseline Flamegraph',
-  comparison: 'Comparison Flamegraph',
+  baseline: 'Baseline',
```

Comment:
```suggestion
  baseline: 'Baseline Time Range',
```

Reply:
Grafana generally uses sentence case (not my preference, but :man_shrugging:)